docs/docs.html

```diff
@@ -1004,6 +1008,145 @@ <h3>Example of Input with validation and error</h3>
+                        style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </section>
+
+                <div class="m-md"></div>
+
+                <section class="example">
+                    <h3>Example of other Text Utilities</h3>
+                    <p>Native UI provides text Utilities such as: small-text, strikethrough, grey-text, lowercase text, uppercase text, capitalise text and marked text.
+                    </p>
+                    <p>To use this alert just use class name alert-box and also add class according to alert type-
+                        alert-primary, alert-secondary, alert-danger, alert-success, alert-warning.
+                        (e.g.class="alert-box alert-primary").
+                        You can copy html part from below code snippet.</p>
+                    <section class="eg-container flex-col-start-start p-md">
+                        <p class="text-lowercase">LOWERCase teXt</p>
+                        <p class="text-uppercase">uppercase text</p>
+                        <div class="text-capitalize">capitalise text</div>
+                        <div class="text-grey">Grey Text</div>
+                        <small>small text</small>
+                        <div class="text-strikethrough">strikethrough text</div>
+                    </section>
+
+                    <div class="m-md"></div>
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28207%2C207%2C207%2C1%29&t=one-light&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=17px&ph=20px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520copy%2520and%2520paste%2520the%2520following%2520code%2520in%2520your%2520html%2520file.%2520Give%2520appropriate%2520class%2520namings%2520as%2520shown%2520in%2520the%2520snippet%2520below%2520--%253E%250A%250A%253Cp%2520class%253D%2522text-lowercase%2522%253ELOWERCase%2520teXt%253C%252Fp%253E%250A%253Cp%2520class%253D%2522text-uppercase%2522%253Euppercase%2520text%253C%252Fp%253E%250A%253Cdiv%2520class%253D%2522text-capitalize%2522%253Ecapitalise%2520text%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522text-grey%2522%253EGrey%2520Text%253C%252Fdiv%253E%250A%253Csmall%253Esmall%2520text%253C%252Fsmall%253E%250A%253Cdiv%2520class%253D%2522text-strikethrough%2522%253Estrikethrough%2520text%253C%252Fdiv%253E"
+  style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```

Comment:
Use prettier or any other code formatter to fix this

Reply:
Yep cool done.